### DISCUSSION TITLE
Mobile Github delete branch when pull request is closed

### DISCUSSION THREAD

**Discussion by Sholde** [community/community#43586] [Mobile] [answered]
Hello, I just saw a pull request I made a long time ago on the mobile application of github, which has been closed in the meantime, but the mobile app has deleted the branch of my pull request.

Link: https://github.com/verificarlo/verificarlo/pull/250

**Comment by stevepopovich** [comment, marked as answer]
Hi @Sholde. We do show a button to delete a branch on closed and merged PRs. The app should not delete the branch on its own, unless your repo is set up to delete branches after merging and you merge on mobile. 

Do you mean that the app deleted the branch by itself?

**Reply by Sholde** [reply]
Hi @stevepopovich,

Yes I mean  that the app delete the branch by itself. Or maybe the button is very sensitive because I never press it directly :/.

But we can restore a branch we have deleted so it is not an end in itself.

**Reply by stevepopovich** [reply]
Ok, we will make sure the button isn't doing anything weird. Thanks for filing!

**Reply by Sholde** [reply]
Ok thanks.
